dependencies/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015-2016 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+FROM openzipkin/jre-full:1.8.0_72
+MAINTAINER OpenZipkin "http://zipkin.io/"
+
+ARG STORAGE_TYPE
+ENV ZIPKIN_REPO https://jcenter.bintray.com
+ENV DEPENDENCIES_VERSION 1.42.1
+
+# Use to set heap, trust store or other system properties.
+ENV JAVA_OPTS -Djava.security.egd=file:/dev/./urandom
+
+WORKDIR /zipkin-dependencies
+
+RUN curl -SL $ZIPKIN_REPO/io/zipkin/zipkin-dependencies-spark-$STORAGE_TYPE/$DEPENDENCIES_VERSION/zipkin-dependencies-spark-$STORAGE_TYPE-$DEPENDENCIES_VERSION-all.jar > zipkin-dependencies.jar
```

Comment:
this url will also change per https://github.com/openzipkin/zipkin-dependencies-spark/issues/31